examples/05_transformers.ipynb

```diff
@@ -0,0 +1,849 @@
+    "* transforming a series into another series. Examples: detrending, smoothing, filtering, lagging.\n",
+    "* transforming a panel into another panel. Examples: principal component projection; applying individual series-to-series transformation to all series in the panel.\n",
+    "* transforming a pair of series into a scalar value. Examples: dynamic time warping distance between series/sequences; generalized alignment kernel between series/sequences.\n",
+    "\n",
+    "Notably, the first three (series to primitive features, series to series, panel to panel) are covered by the same base class template and module. We call these transformers \"time series transformers\", or, simply, \"transformers\".\n",
+    "Kernels and distances for time series and sequences have the same mathematical signature and differ only in mathematical properties (e.g., definiteness assumptions) - they are covered by the more abstract scientific type of \"pairwise transformer\".\n",
+    "\n",
+    "Below, we give an overview in sub-sections:\n",
+    "* reviewing common data container formats for series and panels\n",
+    "* showcasing the signature of time series transformers, that transform a single series or panel\n",
+    "* showcasing the signature of pairwise transformers, that transform a pair of series to a scalar, e.g., distances or kernels\n",
+    "* how to search `sktime` for transformers of a certain type"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1 Data container format<a class=\"anchor\" id=\"section_1_1\"></a>\n",
+    "\n",
+    "`sktime` transformers apply to individual time series and panels. Panels are collections of time series, and we refer to each time series in a panel as an \"instance\" of the panel.\n",
+    "This is formalized as abstract \"scientific types\" `Series` and `Panel`, with multiple possible in-memory representations, so-called \"mtypes\".\n",
+    "\n",
+    "For the purpose of this tutorial, we will be working with the most common mtypes. For more details and formal data type specifications, see the \"datatypes and datasets\" tutorial.\n",
+    "\n",
+    "`Series` are commonly represented as:\n",
+    "\n",
+    "* `pandas.Series` for univariate time series and sequences\n",
+    "* `pandas.DataFrame` for uni- or multivariate time series and sequences\n",
+    "\n",
+    "The `Series.index` and `DataFrame.index` are used for representing the time series or sequence index. `sktime` supports pandas integer, period and timestamp indices.\n",
+    "\n",
+    "`Panel`-s are commonly represented as:\n",
+    "* a `pandas.DataFrame` in a specific format, defined by the `pd-multiindex` mtype. This has a 2-level index, for time points and instances\n",
+    "* a `list` of `pandas.DataFrame`, where all `pandas.DataFrame` are in the `Series` format. The different `list` elements are the different instances\n",
```

Comment:
This is rather unclear since `Series` is ambiguously used, also it is unclear what a `DataFrame` is the format of `Series` would be or how to convert it.